src/garage/_functions.py

```diff
@@ -0,0 +1,49 @@
+"""Functions exposed directly in the garage namespace."""
+from dowel import tabular
+import numpy as np
+
+from garage.misc.tensor_utils import discount_cumsum
+
+
+def log_performance(itr, batch, discount, prefix='Evaluation'):
```

Comment:
should we keep propagating `prefix=` args, or expect people to use `tabular.prefix`?

Reply:
Honestly, I do think propagating `prefix=` args is kinda an anti-pattern, although it is more concise than using `tabular.prefix` (at least in the cases so far). It's convenient for now and it's certainly better than what preceded it (manually implementing tabular prefixes repeatedly), so I say we keep it for the time being.